Dockerfile

```diff
@@ -3,10 +3,12 @@ FROM node:18.18-bullseye-slim as base
 
 ARG BUILD_NUMBER=1_0_0
 ARG GIT_REF=not-available
+ARG GIT_BRANCH=main
```

Comment:
might be better to default this to `not-available`? it'd be more obvious if its not been set